nemo/collections/llm/api.py

```diff
@@ -995,7 +999,7 @@ def export_ckpt(
         if output_path.exists() and not overwrite:
             raise FileExistsError(f"Output path {output_path} exists. Use overwrite=True to force overwrite.")
 
-    output = io.export_ckpt(path, target, output_path, overwrite, load_connector, **kwargs)
+    output = io.export_ckpt(path, target, output_path, overwrite, load_connector, modelopt_export_kwargs, **kwargs)
```

Comment:
FYI @cuichenx @marcromeyn @suiyoubi a change to `export_ckpt` is being pursued here (a useful one)

Reply:
Also FYI @oyilmaz-nvidia